content/community/meetups.md

```diff
@@ -22,8 +22,14 @@ Do you have a local React.js meetup? Add it here! (Please keep the list alphabet
 ## Brazil {#brazil}
 * [Belo Horizonte](https://www.meetup.com/reactbh/)
 * [Curitiba](https://www.meetup.com/pt-br/ReactJS-CWB/)
+* [Florianópolis](https://www.meetup.com/pt-br/ReactJS-Floripa/)
+* [Goiânia](https://www.meetup.com/pt-br/React-Goiania/)
 * [Joinville](https://www.meetup.com/pt-BR/React-Joinville/)
+* [Juiz de Fora](https://www.meetup.com/pt-br/ReactJS-Juiz-de-Fora/)
```

Comment:
O grupo que você está procurando não existe.

🙄

Reply:
https://www.meetup.com/pt-BR/React-Juiz-de-Fora/

Reply:
All checked and fixed!